index.bs

```diff
@@ -5692,14 +5692,27 @@ useful when there is instance data to track.
 The class would be used in code like:
 
 <pre><code class="lang-javascript">
+  function textDecoderStream() {
```

Comment:
I would much rather use the TextDecoderStream class that is planned to be added to the encoding standard (https://github.com/whatwg/encoding/issues/72), even though it isn't available yet. It avoids spending a lot of space on this mostly-unrelated topic, and avoids data corruption when characters are split between chunk boundaries.